std/algorithm/iteration.d

```diff
@@ -2868,6 +2872,80 @@ private template ReduceSeedType(E)
     }
 }
 
+
+/++
+Implements the homonym function (also known as $(D accumulate), $(D
```

Comment:
Would be nice to use backticks instead of `$(D ...)` throughout (not a blocker).